pkg/services/ngalert/sender/sender.go

```diff
@@ -127,13 +127,14 @@ func (s *ExternalAlertmanager) ApplyConfig(orgId, id int64, alertmanagers []Exte
 }
 
 func (s *ExternalAlertmanager) Run() {
+	logger := s.logger
```

Comment:
`ngalert/remote.NewAlertmanager` calls `Run` and then `ApplyConfig`, which causes a race condition between the two trying to access `logger`, so I created a local copy.